source/admin_manual/list_of_events.rst

```diff
@@ -2333,6 +2333,38 @@ happens.
 | object_id             | Object ID in the storage                             |
 +-----------------------+------------------------------------------------------+
 
+fs_dictmap_max_bucket_changed
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 2.3.13
+
+This event is sent whenever the ``max_bucket`` value for a mailbox changes.
+There can be two situations this happens. Either a new mail is added to a
+mailbox, where the current bucket is found to be filled and a next bucket is
+started to get filled (``reason == file``).
```

Comment:
I think "the" next bucket?.. and "be" filled?..

Reply:
And I think ``reason=file`` is clearer since our event filter query uses one `=`